### PR TITLE
thumbnail

### PR DESCRIPTION
**Pull request by PRASHANT1104**
## Description
Thumbnail support for Result component 

## List of changes
packages/react-search-ui/src/containers/Result.js - Added code to handle Result component , if thumbnail is provided .
examples/sandbox/src/App.js (for sandbox) - If thumbnail is provided , please uncomment the line 76 (  //  image_url:{raw:{}} ) to render the thumbnail .

## Associated Github Issues
#425

### CODE REVIEW

**Comment by JasonStoltz** [comment]
This is an awesome PR, thank you so much. We're going to submit a PR to your branch with a few updates. I'm going to add storybook stories and fix the broken spec. We also might tweak your CSS a bit. Could you please sign the CLA in the meantime?

**Comment by JasonStoltz** [comment]
latest

The main thing I want to solve is make sure that the thumbnail maintains it's proportions.

File: examples/sandbox/src/App.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ const config = {
       nps_link: { raw: {} },
       states: { raw: {} },
       date_established: { raw: {} },
+    //  image_url:{raw:{}},
```
Comment (JasonStoltz) [inline examples/sandbox/src/App.js]: I just added image_url to this data set so that this should work.

File: examples/sandbox/src/App.js [hunk 2]
```diff
@@ -238,6 +240,7 @@ export default function App() {
                     <Results
                       titleField="title"
                       urlField="nps_link"
+                      thumbnailField="image_url"
```
Comment (JasonStoltz) [inline examples/sandbox/src/App.js]: You'll need to add support for the `thumbnailField` in the following files:

packages/react-search-ui/src/containers/Results.js
packages/react-search-ui/src/containers/Result.js

File: examples/sandbox/src/App.js [hunk 3]
```diff
@@ -256,4 +259,4 @@ export default function App() {
       </WithSearch>
     </SearchProvider>
   );
-}
+}
```
Comment (JasonStoltz) [inline examples/sandbox/src/App.js]: Re-add this return please.

File: examples/sandbox/src/App.js (hunk 1 above)
Comment (JasonStoltz) [reply examples/sandbox/src/App.js]: Also, make sure the formatting of this line matches the formatting of the other lines, including spaces.

**Comment by PRASHANT1104** [comment]
@JasonStoltz I have made the changes . Can you review it ?

**Comment by JasonStoltz** [comment]
This is looking good. I've asked a colleague to take a look at the CSS to see if we can't get images rendering in a consistent size.

**Comment by constancecchen** [comment]
@JasonStoltz https://github.com/PRASHANT1104/search-ui/pull/2 - let me know if this is what you meant by rendering in a consistent size? If not, I can continue to tweak.

**Comment by JasonStoltz** [comment]
@constancecchen Yep, thanks.

**Comment by JasonStoltz** [comment]
@PRASHANT1104 This is close. Can you merge @constancecchen's PR? We also need to add a `thumbnailAltField` to support populating the `alt` tag of the image, as @constancecchen mentioned. If you can handle that in this PR, that would be great. Otherwise, we can add it in a follow up PR.

**Comment by JasonStoltz** [comment]
Still important.

**Comment by ballabusta** [comment]
Could we pass that new thumbnail to the autocompleteResults as well ?

**Comment by HesterG** [comment]
Is this gonna merge? I found it would be very useful to a project i am working on.

**Comment by JasonStoltz** [comment]
I might be able to get this merged. Let me take a second look and see if I can get this done today or tomorrow.

**Comment by JasonStoltz** [comment]
Taking over this PR and moving it to 566

**Pull request closed without merging**